apps/api/src/app/portfolio/portfolio.service.ts

```diff
@@ -1652,7 +1654,19 @@ export class PortfolioService {
       case 'ytd':
         portfolioStart = max([
           portfolioStart,
-          setDayOfYear(new Date().setHours(0, 0, 0, 0), 1)
+          startOfYear(new Date().setHours(0, 0, 0, 0))
```

Comment:
here I took the liberty of replacing `setDayOfYear(..., 1)` by `startOfYear(...)` which is a bit more explicit IMHO.

https://date-fns.org/v3.3.0/docs/startOfYear